### DISCUSSION TITLE
Kubernetes sql_exporter deployment

### DISCUSSION THREAD

**Discussion by praveen-kumar-yadav-db** [burningalchemist/sql_exporter#264] [General]
Is there any sample yaml available which can be deployed on kubernetes environment. I am trying below one but it is unable to find the sql_exporter.yml.

kind: Deployment
apiVersion: apps/v1
metadata:
  name: prom-sql-exporter
spec:
  replicas: 1
  selector:
    matchLabels:
      app: prom-sql-exporter
  template:
    metadata:
      creationTimestamp: null
      labels:
        app: prom-sql-exporter
      annotations:
        prometheus.io/port: '9237'
        prometheus.io/scrape: 'true'
    spec:
      volumes:
        - name: volume-0nf3v
          configMap:
            name: sql-exporter-config
            defaultMode: 420
      containers:
        - resources:
            limits:
              cpu: 250m
              memory: 32Mi
            requests:
              cpu: 5m
              memory: 16Mi
          terminationMessagePath: /dev/termination-log
          name: prom-sql-exporter
          ports:
            - name: http
              containerPort: 9237
              protocol: TCP
          imagePullPolicy: IfNotPresent
          volumeMounts:
            - name: volume-0nf3v
              mountPath: /config
          terminationMessagePolicy: File
          image: burningalchemist/sql_exporter:0.11
      restartPolicy: Always
      terminationGracePeriodSeconds: 30
      dnsPolicy: ClusterFirst
      securityContext: {}
      schedulerName: default-scheduler

**Comment by burningalchemist** [comment]
Hey @praveen-kumar-yadav-db, you need to attach `sql_exporter.yml` as a volume yourself, this file is required to run `sql_exporter`.

Alternatively, you might want to try out the helm chart:
```
helm repo add https://burningalchemist.github.io/sql_exporter/
helm repo update
```

You can check the default values here: https://github.com/burningalchemist/sql_exporter/tree/master/helm
